pkgs/tools/text/grip-search/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, boost, pkgconfig, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "grip-search";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner = "sc0ty";
+    repo = "grip";
+    rev = "v${version}";
+    sha256 = "0bkqarylgzhis6fpj48qbifcd6a26cgnq8784hgnm707rq9kb0rx";
+  };
+
+  nativeBuildInputs = [ pkgconfig cmake ];
```

```suggestion
  nativeBuildInputs = [ pkgconfig cmake catch2 ];
  
  doCheck = true;
```

Please verify, but I think this should enable the test suite. At least that's what was done in #72893